src/shared/services/HttpService.ts

```diff
@@ -65,10 +63,6 @@ class WrappedLemmyHttpClient {
               state: !(res === undefined || res === null) ? "success" : "empty",
             };
           } catch (error) {
-            if (!silent) {
-              console.error(`API error: ${error}`);
-              toast(I18NextService.i18n.t(error), "danger");
-            }
```

Comment:
Displaying errors should probably be handled by the places that call the client so that error toasts don't pop up unexpectedly.